doc/build-test.md

```diff
@@ -25,12 +25,12 @@ Our builds fall into these categories:
 Test
 ----
 
-Our C# tests use [xUnit](https://xunit.github.io/docs/getting-started-dotnet-core). Each test is tagged with one of the following attributes:
+Our C# tests use [xUnit](https://xunit.net/docs/getting-started/netcore/cmdline) for almost all test assemblies except Microsoft.Azure.Devices.Edge.Test where used [NUnit](https://github.com/nunit/docs/wiki). Each XUnit test is tagged with one of the following attributes:
```

Comment:
```diff
-where used [NUnit](https://github.com/nunit/docs/wiki).
+where [NUnit](https://github.com/nunit/docs/wiki) is used.
```

Reply:
The distinction is that unit tests and integration tests use xUnit, but our **new** end-to-end tests use NUnit. The NUnit tests (under /test) will eventually replace everything under /smoke (which this markdown file mentions a little farther down).